Source/Scene/GlobeTranslucencyFramebuffer.js

```diff
@@ -39,110 +30,70 @@ function GlobeTranslucencyFramebuffer() {
-    !globeTranslucency._colorTexture.isDestroyed() &&
-    globeTranslucency._colorTexture.destroy();
-  globeTranslucency._depthStencilTexture =
-    globeTranslucency._depthStencilTexture &&
-    !globeTranslucency._depthStencilTexture.isDestroyed() &&
-    globeTranslucency._depthStencilTexture.destroy();
-  globeTranslucency._depthStencilRenderbuffer =
-    globeTranslucency._depthStencilRenderbuffer &&
-    !globeTranslucency._depthStencilRenderbuffer.isDestroyed() &&
-    globeTranslucency._depthStencilRenderbuffer.destroy();
-  globeTranslucency._framebuffer =
-    globeTranslucency._framebuffer &&
-    !globeTranslucency._framebuffer.isDestroyed() &&
-    globeTranslucency._framebuffer.destroy();
-  globeTranslucency._packedDepthTexture =
-    globeTranslucency._packedDepthTexture &&
-    !globeTranslucency._packedDepthTexture.isDestroyed() &&
-    globeTranslucency._packedDepthTexture.destroy();
-  globeTranslucency._packedDepthFramebuffer =
-    globeTranslucency._packedDepthFramebuffer &&
-    !globeTranslucency._packedDepthFramebuffer.isDestroyed() &&
-    globeTranslucency._packedDepthFramebuffer.destroy();
+  globeTranslucency._framebuffer.destroyResources();
+  globeTranslucency._packedDepthFramebuffer.destroyResources();
 }
 
-function createResources(globeTranslucency, context, width, height, hdr) {
-  var pixelDatatype = hdr
-    ? context.halfFloatingPointTexture
-      ? PixelDatatype.HALF_FLOAT
-      : PixelDatatype.FLOAT
-    : PixelDatatype.UNSIGNED_BYTE;
-  globeTranslucency._colorTexture = new Texture({
-    context: context,
-    width: width,
-    height: height,
-    pixelFormat: PixelFormat.RGBA,
-    pixelDatatype: pixelDatatype,
-    sampler: Sampler.NEAREST,
-  });
-
-  if (context.depthTexture) {
-    globeTranslucency._depthStencilTexture = new Texture({
-      context: context,
-      width: width,
-      height: height,
-      pixelFormat: PixelFormat.DEPTH_STENCIL,
-      pixelDatatype: PixelDatatype.UNSIGNED_INT_24_8,
-    });
-  } else {
-    globeTranslucency._depthStencilRenderbuffer = new Renderbuffer({
-      context: context,
-      width: width,
-      height: height,
-      format: RenderbufferFormat.DEPTH_STENCIL,
-    });
-  }
-
-  globeTranslucency._framebuffer = new Framebuffer({
-    context: context,
-    colorTextures: [globeTranslucency._colorTexture],
-    depthStencilTexture: globeTranslucency._depthStencilTexture,
-    depthStencilRenderbuffer: globeTranslucency._depthStencilRenderbuffer,
-    destroyAttachments: false,
-  });
-
-  globeTranslucency._packedDepthTexture = new Texture({
-    context: context,
-    width: width,
-    height: height,
-    pixelFormat: PixelFormat.RGBA,
-    pixelDatatype: PixelDatatype.UNSIGNED_BYTE,
-    sampler: Sampler.NEAREST,
-  });
-
-  globeTranslucency._packedDepthFramebuffer = new Framebuffer({
-    context: context,
-    colorTextures: [globeTranslucency._packedDepthTexture],
-    destroyAttachments: false,
-  });
-}
+// var pixelDatatype = hdr
```

Comment:
This needs to be removed, just commenting it out for now in case I move it back (see https://github.com/CesiumGS/cesium/pull/9966/files#r764464629).